defaults.yml

```
@@ -194,6 +194,15 @@ categories:
             name: com.apple.dock autohide
             value: true
         after: killall Dock
+      - key: static-only
+        domain: com.apple.dock
+        title: Only show active applications in Dock
+        param:
+          type: bool
+        examples:
+          - value: true
+            default: true
+        after: killall Dock
```

Add description and examples details

```suggestion
      - key: static-only
        domain: com.apple.dock
        title: Active applications only
        description: Only show opened apps in Dock
        param:
          type: bool
        examples:
          - value: true
             text: Only show active apps
          - value: false
             default: true
             text: Hide inactive apps from the Dock
        after: killall Dock
```